projects/1.19/assets/the-masquerade-illager-boss/masquerader_mod/lang/zh_cn.json

```
@@ -0,0 +1,37 @@
+{
+  "entity.masquerader_mod.masquerader": "假面愚者",
+  "entity.masquerader_mod.masquerader_clone": "假面愚者",
+  "entity.masquerader_mod.masked_vindicator": "卫道士",
+  "entity.masquerader_mod.masked_evoker": "唤魔者",
+  "entity.masquerader_mod.masked_illusioner": "幻术师",
+  "entity.masquerader_mod.masked_witch": "女巫",
+  "entity.masquerader_mod.masked_villager": "戴面具的村民",
+
+  "sounds.masquerader_mod.masquerader_change_mask": "假面愚者：准备更换面具",
+  "sounds.masquerader_mod.masquerader_laugh": "假面愚者：大笑",
+  "sounds.masquerader_mod.masquerader_prepare_clone": "假面愚者：准备幻术伏击",
+  "sounds.masquerader_mod.masquerader_death": "假面愚者：死亡",
+  "sounds.masquerader_mod.clone_death": "假面愚者分身：消失",
+
+  "advancements.masquerader_mod.invitation_advancement": "化装舞会！",
+  "advancements.masquerader_mod.invitation_advancement_desc": "受邀参加一场高档社交晚会",
+  "advancements.masquerader_mod.ballroom_advancement": "“翩翩”来迟",

```

```suggestion
  "advancements.masquerader_mod.ballroom_advancement": "“珊珊”来迟",
```
还是用读音一致的吧，“翩翩”有点牛头不对马嘴